test/functional/feature_fee_estimation.py

```diff
@@ -278,12 +283,16 @@ def sanity_check_rbf_estimates(self, utxos):
             while len(utxos_to_respend) > 0:
                 u = utxos_to_respend.pop(0)
                 tx = make_tx(self.wallet, u, high_feerate)
+                self.confutxo.append({
+                    "txid": tx["txid"],
+                    "vout": 0,
+                    "value": Decimal(tx["tx"].vout[0].nValue) / COIN
+                })
```

Comment:
Wondering why these calls are not required after every `make_tx` call?

Reply:
Because `make_tx` is only constructing the transaction, the transaction created might not be confirmed, so Its inaccurate to add it to confirmed utxo's after constructing the transaction.

Transactions added to the confirmed utxo's in the diff (`sanity_check_rbf_estimates`) are all going to be confirmed eventually so we are accounting for them.